src/components/pod/pod.test.js

```diff
@@ -13,6 +55,234 @@ const PodWithSoftDelete = () => (
+      (variant, color, boxShadow) => {
+        CypressMountWithProviders(<PodComponent variant={variant} />);
+        divPodComponent()
+          .should("have.css", "back-ground", color)
+          .and("have.css", "box-shadow", boxShadow);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check title as %s for Pod component",
+      (title) => {
+        CypressMountWithProviders(<PodComponent title={title} />);
+        podTitle().should("have.text", title);
+      }
+    );
+    it.each(specialCharacters)(
+      "should check subtitle as %s for Pod component",
+      (subtitle) => {
+        CypressMountWithProviders(<PodComponent subtitle={subtitle} />);
+        podSubTitle().should("have.text", subtitle);
+      }
+    );
+
+    it.each(["left", "center", "right"])(
+      "should check when title alignment is %s for Pod component",
+      (alignTitle) => {
+        CypressMountWithProviders(<PodComponent alignTitle={alignTitle} />);
+        podTitle().should("have.css", "text-align", alignTitle);
+      }
+    );
+
+    it.each(specialCharacters)(
+      "should check footer as %s for Pod component",
+      (footerText) => {
+        CypressMountWithProviders(<PodComponent footer={footerText} />);
+        podFooter().should("have.text", footerText).and("be.visible");
+      }
+    );
+
+    it.each(["100px", "200px", "300px"])(
+      "should check when internalEditButton is %s for Pod component",
+      (height) => {
+        CypressMountWithProviders(<PodComponent height={height} />);
+        podComponent().should("have.css", "height", height);
+      }
+    );
+
+    it.each([
+      [false, "not.exist"],
+      [true, "be.visible"],
+    ])(
+      "should check when softDelete is %s for Pod component",
+      (boolVal, state) => {
+        CypressMountWithProviders(
+          <SoftDeletePodComponent softDelete={boolVal} />
+        );
+        podSoftDelete().should(state);
+      }
+    );
+
+    it.each([
+      [false, "77.5625px"],
+      [true, "1292px"],
+    ])(
+      "should check when editContentFullWidth is %s for Pod component",
+      (boolVal, width) => {
+        CypressMountWithProviders(
+          <PodComponent editContentFullWidth={boolVal} />
+        );
+        divPodComponent().should("have.css", "width", width);
+      }
+    );
+
+    it.each([true, false])(
+      "should check when displayEditButtonOnHover is %s for Pod component",
+      (boolVal) => {
+        CypressMountWithProviders(
+          <PodComponent displayEditButtonOnHover={boolVal} />
+        );
+        if (boolVal === true) {
+          podContent().click();
+          podEditContainer().should("be.visible");
+          podDelete().should("not.be.visible");
+        } else {
+          podDelete().should("be.visible");
+        }
+      }
+    );
+
+    it.each([
+      [true, "1350px"],
+      [false, "77.5625px"],
+    ])(
+      "should check when internalEditButton is %s for Pod component",
```

Comment:
same as above "should check the width value when internalEditButton is %s for Pod component" something like this